server/src/main/java/io/spine/system/server/SystemRequestFactory.java

```diff
@@ -59,23 +59,22 @@ static ActorRequestFactory instance(boolean multitenant) {
     }
 
     private static ActorRequestFactory newForCurrentTenant() {
-        TenantFunction<ActorRequestFactory> createFactory =
-                new TenantFunction<ActorRequestFactory>(true) {
-                    @Override
-                    public ActorRequestFactory apply(@Nullable TenantId tenantId) {
-                        checkNotNull(tenantId);
-                        return newFactoryFor(tenantId);
-                    }
-                };
-        ActorRequestFactory result = createFactory.execute();
-        checkNotNull(result);
+        var createFactory = new TenantFunction<ActorRequestFactory>(true) {
+            @Override
+            public ActorRequestFactory apply(@Nullable TenantId tenantId) {
```

Comment:
Please consider removing `@Nullable` annotation and reverting to the `checkNotNull()`.